Tests/Unit/EmogrifierTest.php

```diff
@@ -2067,4 +2067,20 @@ public function emogrifierNotMapsCssToHtmlIfFeatureIsNotEnabled()
             $html
         );
     }
+ 
+    /**
+     * @test
+     */
+    public function emogrifierMapsClassWithLineFeedInAttrib()
```

Comment:
We should test all cases (preferably with a data provider):

- leading space
- trailing space
- leading NL
- trailing NL
- leading tab
- trailing tab
- leading CRLF
- trailing CRLF